--- cpu2006/config/Example-linux64-amd64-gcc43+.cfg	2011-08-06 03:59:19.000000000 +0800
+++ cpu2006/config/config.cfg	2019-01-09 09:49:21.147072961 +0800
@@ -127,8 +127,8 @@ notes_os_000 ='ulimit -s unlimited' was
 #####################################################################
 
 default=base=default=default:
-COPTIMIZE   = -O2 -fno-strict-aliasing
-CXXOPTIMIZE = -O2 -fno-strict-aliasing
+COPTIMIZE   = -O2 -fno-strict-aliasing -std=gnu89
+CXXOPTIMIZE = -O2 -fno-strict-aliasing -std=c++03
 FOPTIMIZE   = -O2 -fno-strict-aliasing
 
 #####################################################################
